apps/judicial-system/web/src/routes/Shared/Cases/Cases.tsx

```diff
@@ -169,7 +172,18 @@ export const Cases: FC = () => {
 
       if (isDistrictCourtUser(user)) {
         if (isIndictmentCase(c.type)) {
-          return !isCompletedCase(c.state)
+          const sentToPublicProsecutor = c.eventLogs?.some(
+            (log) =>
+              log.eventType === EventType.INDICTMENT_SENT_TO_PUBLIC_PROSECUTOR,
+          )
+          const isFineOrRuling =
+            c.indictmentRulingDecision === CaseIndictmentRulingDecision.FINE ||
+            c.indictmentRulingDecision === CaseIndictmentRulingDecision.RULING
+
+          return (
+            !isCompletedCase(c.state) ||
```

Comment:
One thought since we are checking also if the case is not completed yet based on the state. Should we not rather include this logic where we are setting if the case state as completed and then only checking `!isCompeltedCase(c.state)`? or would that result in unexpected tags?

Reply:
Good point. I feel like we should maybe not do that, since we are checking for completed cases in soooo many other places and perhaps this condition doesn't apply there also.

Holla if you strongly disagree 👀 

<img width="365" alt="Screenshot 2025-02-14 at 15 05 23" src="https://github.com/user-attachments/assets/fafddc8a-ceba-4d56-b2ce-7973404a8585" />